src/index.js

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
```

Comment:
Prefer to leave this explicit React import in index.js in case someone else running your project is working w/ an older version of React that does need this import statement.